client/components/EditorMenu/ScenarioStatusMenuItem.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import changeCase from 'change-case';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+
+let cachedStatusOptions = null;
+
+class ScenarioStatusMenuItem extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            statusOptions: null
+        };
+
+        this.fetchStatusOptions();
+    }
+
+    async fetchStatusOptions() {
+        const statusOptions = await (cachedStatusOptions ||
+            (await fetch('/api/status')).json());
+
+        statusOptions.forEach(
+            statusOption =>
+                (statusOption.name = changeCase.titleCase(statusOption.name))
```

Comment:
This is cool! We can use this in the slide buttons as well when users are running scenarios.